addon/mixins/application-route-mixin.js

```diff
@@ -58,6 +58,10 @@ export default Mixin.create({
     return fastboot ? fastboot.get('isFastBoot') : false;
   }),
 
+  _cookies: computed(function() {
+    return getOwner(this).lookup('service:cookies');
+  }),
```

Comment:
I had `_cookies` and `_secureCookies` as members strictly for mocking in tests.  Is there a reference in the test suite for mocking the lookup function?